frontend/src/App.tsx

```diff
@@ -39,16 +40,24 @@ const App = () => {
           </Snackbar>
           <CssBaseline />
           <Header authSession={authSession} />
-          <Routes>
-            <Route path='/' element={<LandingPage authSession={authSession} />} />
-            <Route path='/Elections' element={<Elections authSession={authSession} />} />
-            <Route path='/Login' element={<Login />} />
-            <Route path='/Debug' element={<DebugPage authSession={authSession} />} />
-            <Route path='/CreateElection' element={<AddElection authSession={authSession} />} />
-            <Route path='/Election/:id/*' element={<Election authSession={authSession} />} />
-            <Route path='/DuplicateElection/:id' element={<DuplicateElection authSession={authSession} />} />
-            <Route path='/Sandbox' element={<Sandbox />} />
-          </Routes>
+          <Box
+            sx={{
+              width: '100%',
+              minHeight: `calc(100vh - 255px - 64px)` //Sets min height of content to 100% minus footer and header in order to force footer to the bottom
+            }}>
+            <Routes>
+              <Route path='/' element={<LandingPage authSession={authSession} />} />
+              <Route path='/Elections' element={<Elections authSession={authSession} />} />
+              <Route path='/Login' element={<Login />} />
+              <Route path='/Debug' element={<DebugPage authSession={authSession} />} />
+              <Route path='/CreateElection' element={<AddElection authSession={authSession} />} />
+              <Route path='/Election/:id/*' element={<Election authSession={authSession} />} />
+              <Route path='/DuplicateElection/:id' element={<DuplicateElection authSession={authSession} />} />
+              <Route path='/Sandbox' element={<Sandbox />} />
+            </Routes>
+
+          </Box>
+          <Footer />
```

Comment:
I was thinking about if we wanted to do a call to action for contributions somewhere in the footer, but I don't think we need one. I elaborated on my thoughts [here](https://github.com/Equal-Vote/star-server/issues/289)